res/games/gameitem.css

```diff
@@ -0,0 +1,117 @@
+{%- macro playermacro(player,teamalign) %}
+        {% if player.login is not defined %}
+            {% set player_login = player %}
+        {% else %}
+            {% set player_login = player.login %}
+        {% endif %}
+        
+        {% set player_country = "" %}
+        {% set player_global_rating = "???" %}
+        {% if player.countrypath is defined %}
+            {% set player_country = player.countrypath %}
+            {% set player_global_rating = (player.global_rating[0]-(player.global_rating[1]*3))|int %}
+        {% endif %}
+
+        {% set width = 130 %}
+        {% if teams[1] is not defined %}
+            {% set alignment = 'left' %}
+            {% if "-" not in player.login %}
+	            {% set width = 0 %}
+	        {% endif %}
+        {% else %}
+            {% set alignment = 'right' %}
+        {% endif %}
+
+        <tr>
+        {% if teamalign == "left" -%}
+            <td align={{ alignment }} width='{{ width }}'>
+            {{ player_login }}
+            </td>
+            <td>
+            <img src='{{ player_country }}'>
+            </td>
+            <td align={{ alignment }} valign='middle' width='10'>
+            ({{ player_global_rating }})
+            </td>
+        {% else %}
+            <td align='left' valign='middle' width='10'>
+            {% if player_global_rating is defined %}
+                ({{ player_global_rating }})
+            {% else %}
+                (???)
+            {% endif %}
+            </td>
+            <td>
+            <img src='{{ player_country }}'>
+            </td>
+            <td align='left' width={{ width }}>
+            {{ player_login }}
+            </td>
+        {% endif %}
+         </tr>
+{% endmacro -%}
+
+
+{%- macro versus_string(fontsize) %}
+    <td align='middle' valign='middle' height='100%'>
+    <font color ='black' size={{ fontsize }}>VS</font>
+    </td>
+{% endmacro -%}
+
+
+{%- macro modtip(mods) %}
+    <b><br/>With:</b><br/>
+    {{"<br />".join(mods)}}
+{% endmacro -%}
+
+
+{%- macro observertooltip(observers) %}
+    <b>Observers:</b>
+    {% for observer in observers %}
+        <br/>{{ observer.login }} <img src='{{ observer.countrypath }}'> ({{ (observer.global_rating[0]-(observer.global_rating[1]*3))|int }})
```

Comment:
im a little confused now though - why make a new attribute for the "displayed_rating", but not for the "countrypath"? but yea, putting the rating into the class is np

Reply:
Mainly because a 'displayed rating' property is something dependent on the 'player' entity only, and a path to an icon that represents the country is theme specific - it's related to client display, not to the player himself.